eva/binder/statement_binder.py

```diff
@@ -75,6 +76,10 @@ def _bind_abstract_expr(self, node: AbstractExpression):
         for child in node.children:
             self.bind(child)
 
+    @bind.register(ExplainStatement)
+    def _bind_explain_statement(self, node: ExplainStatement):
+        self.bind(node.explainable_stmt)
```

Comment:
Right now, bind will create catalog entries. @gaurav274 This can be a consideration to move catalog operations to executors.